dist/docs/scripts/app.js

```diff
@@ -88,11 +92,58 @@
         msmNotification.clearAll();
       };
 
-      /**
-       * Main method
-       */
       (function initController() {
         $log.debug('[NotificationController] Initializing...');
       })();
     }
+
+    function ModalController($scope, $log, $modal) {
+      $scope.items = ['Example item 1', 'Example item 2', 'Example item 3'];
+
+      $scope.open = function (size) {
+        var modalInstance = $modal.open({
+          animation: true,
+          templateUrl: 'components/ui/msm-modal/modal.tpl.html',
```

Comment:
We tend to not suffix templates with `.tpl.html` as all html files in the components directory are templates. I'll rename this for mindsmash-ui.